### PR TITLE
Proposed Changes

### PR DESCRIPTION
**Pull request by RCMorea**
These all work on my system.  Assuming my git-fu has proved sufficient, perhaps you can see them?

1) Added “CURRENT ACTOR” label to primary marque.
2)  Corrected defect in Shift-R behavior.  Now all characters are displayed as “available” upon reset.
3)  Added ‘h’ as well as ‘?’ to commands which display the command key reference since this behavior is specified in command key reference.
4)  Edited command key reference to reflect ‘=’ instead of ‘[Shift] + =’ as trigger for setting next empty initiative.  This reflects program behavior.
5)  Corrected defect in move actor behavior.  Actors will not receive initiative of NaN when moved behind actors with no set initiative.
6)  Future:  Create command to make all actors permanent rather than one at a time.
7)  Future:  Fix “clear initiative” function which currently is Mac-specific.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -607,7 +611,8 @@ $(function(){
     },
 
     resetActorFeatures: function(actor) {
-      actor.removeTransientFeatures();
+      //actor.removeTransientFeatures();
+      actor.resetFeatures();
```
Comment (eephillip) [inline js/app.js]: see resetFeatures

File: js/models/actor.js [hunk 2]
```diff
@@ -116,12 +116,18 @@ var Actor = Backbone.Model.extend({
...
+  //     return (f != 'persistent');
+  //   });
+  //   this.save({
+  //     'features': newFeatures
+  //   });
+  // },
+
+  resetFeatures: function() {
+    var newFeatures = ['available'];
+    this.save({'features': newFeatures
```
Comment (eephillip) [inline js/models/actor.js]: The intent of the `<shift> + r`  `Reset Session` was to do the following.
1. Clear all conditions not marked `Persist last condition` see the 'o' command
  Visualized by underlined text.
2. If the the actor itself is not marked as `persistent`  see `<shift> + p`
  Remove the actor entirely. 

As far as the `available` feature or it's companion `reacted`, I really don't have a preference for setting those on reset. 

However, if the check for persistence isn't maintained the entire actor list would be removed entirely on two back to back resets.

 Lets say you have 8 primary members all marked and persistent members, which you should only have to do once. And then a battle ensues with 10 additional Kobolds. You would enter all 10 as non persistent. Once out of initiative, you can reset the entire list, maintaining primary members, and trimming out the rest with one command.

File: js/app.js [hunk 3]
```diff
@@ -482,8 +483,7 @@ $(function(){
       target_index = candidate_index < 0 ? 0 : candidate_index;
       if (target_index != current_index) {
         target_initiative = parseInt(Actors.at(target_index).get('order')) + 1;
-        Actors.at(current_index).save({'order': target_initiative});
-      }
```
Comment (eephillip) [inline js/app.js]: lost a return

File: index.html [hunk 4]
```diff
@@ -42,7 +42,9 @@
   </div>
 
   <div class="panel fixed" id="fixed-panel">
-    <div class='marquee primary' id="marquee-primary"></div>
+    <div class='marquee primary' id="marquee-primary">
+      <p style="margin: 8px 0 0 22px;">CURRENT ACTOR</p>
+    </div>
```
Comment (eephillip) [inline index.html]: I like it

File: index.html [hunk 5]
```diff
@@ -244,7 +246,7 @@ <h2>Command Keys Reference</h2>
         <tr class="break"><td colspan=2><b>Getting Started</b></td></tr>
         <tr><th>Key</th><th class="behavior">Behavior</th></tr>
           <tr><td><b>[Shift] + A</b></td><td>Add an actor</td></tr>
-          <tr><td><b>[Shift] + =</b></td><td>Set next empty initiative</td></tr>
+          <tr><td><b>=</b></td><td>Set next empty initiative</td></tr>
```
Comment (eephillip) [inline index.html]: good find

File: js/app.js [hunk 6]
```diff
@@ -493,7 +493,11 @@ $(function(){
       if (target_index != current_index) {
         candidate_initiative = parseInt(Actors.at(target_index).get('order')) - 1;
         target_initiative = candidate_initiative < 0 ? 0 : candidate_initiative;
-        Actors.at(current_index).save({'order': target_initiative});
+        if (isNaN(target_initiative)) {
+          return;
+          }
+        else {
+          Actors.at(current_index).save({'order': target_initiative});}
```
Comment (eephillip) [inline js/app.js]: Nice catch, Don't forget `actorUp: function`
Suggestion, though es6
```
if (Number.isInteger(target_initiative)) {
  Actors.at(current_index).save({'order': target_initiative});
}
```

**Comment by RCMorea** [comment]
I am going to close this and resubmit each change separately.  I will take eephillips comments into account when I resubmit.

**Pull request closed without merging**